packages/core-flows/src/api-key/workflows/delete-api-keys.ts

```diff
@@ -7,6 +9,10 @@ export const deleteApiKeysWorkflowId = "delete-api-keys"
 export const deleteApiKeysWorkflow = createWorkflow(
   deleteApiKeysWorkflowId,
   (input: WorkflowData<WorkflowInput>): WorkflowData<void> => {
-    return deleteApiKeysStep(input.ids)
+    deleteApiKeysStep(input.ids)
+
+    removeRemoteLinkStep({
```

Comment:
NIT: Might be good to mention that the API keys might not be `publishable` keys (although there is no harm in running this command). Alternatively we can fetch before deleting, but I think it's not really necessary